pkg/envoy/api/http_connection_manager_test.go

```diff
@@ -40,7 +40,7 @@ func TestNewHTTPConnectionManagerWithoutAccessLogWithoutProxyProtocol(t *testing
 	}
 	connManager := NewHTTPConnectionManager("test", &kourierConfig)
 	assert.Check(t, len(connManager.AccessLog) == 0)
-	assert.Check(t, connManager.UseRemoteAddress == nil)
+	assert.Check(t, connManager.UseRemoteAddress.Value == false)
```

Comment:
Are they mutual exclusive? Should we warn the user? :thinking:

Reply:
What do you mean by that?

Reply:
I mean can we use proxy protocol along with user-remote-address, what if both are enabled?

Reply:
It does not seem to create a problem but don't have an environment to easily check this.